### PR TITLE
Installation instructions

### PR DESCRIPTION
**Pull request by binarybottle**

### CODE REVIEW

**Comment by binarybottle** [comment]
I was able to create a package and install it on my Android device.  I also have Girder up and running on AWS (EC2 + S3).  I have not tested whether data collected from the app saves to Girder.

File: INSTALL [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+#-------------------------------------------------------------------------------
+ionic upload  # after creating ionic account
+ionic share EMAIL
+
+#-------------------------------------------------------------------------------
+# Package and run on a device or simulator
+#-------------------------------------------------------------------------------
+ionic upload  # after creating ionic account
+ionic package build android --profile dev
+cordova platform add android
```
Comment (satra) [inline INSTALL]: this step is unnecessary. the `ionic state reset` command earlier creates both the android and ios platforms.

**Comment by satra** [comment]
@binarybottle - thank you. will start trimming this repo to a minimal demo framework.

File: INSTALL (hunk 1 above)
Comment (binarybottle) [reply INSTALL]: Okay -- I replaced these three commands with the ionic state reset command.

**Pull request merged**